tracing/Cargo.toml

```diff
@@ -71,6 +71,3 @@ harness = false
 [badges]
 azure-devops = { project = "tracing/tracing", pipeline = "tokio-rs.tracing", build = "1" }
 maintenance = { status = "actively-developed" }
-
-[target.'cfg(not(feature = "std"))'.dependencies]
-spin = "0.5"
```

Comment:
`tracing` re-exports the `spin::Once` type in the macro support module, here:
https://github.com/tokio-rs/tracing/blob/773c44054aed7f072ff03ee5dc64391f799dc02d/tracing/src/lib.rs#L703-L704

In order for the macros to continue working, we'll need to reexport the vendored `spin::Once` from `tracing-core`. So, I think that re-export will need to be changed to:
```rust

    #[cfg(not(feature = "std"))]
    pub type Once = tracing_core::Once<()>;
```